Jenkinsfile.baguette

```diff
@@ -8,64 +8,77 @@ pipeline {
-                            sh 'make -f docker.Makefile lint'
-                            sh 'make -f docker.Makefile cross e2e-cross tars'
-                            dir('bin') {
-                                stash name: 'binaries'
-                            }
-                            dir('e2e') {
-                                stash name: 'e2e'
-                            }
-                            dir('examples') {
-                                stash name: 'examples'
-                            }
-                            if(!(env.BRANCH_NAME ==~ "PR-\\d+")) {
-                                stash name: 'artifacts', includes: 'bin/*.tar.gz', excludes: 'bin/*-e2e-*'
-                                archiveArtifacts 'bin/*.tar.gz'
+            parallel {
+                stage('Binaries'){
+                    agent {
+                        label 'team-local && linux'
+                    }
+                    steps  {
+                        dir('src/github.com/docker/app') {
+                            script {
+                                try {
+                                    checkout scm
+                                    sh 'make -f docker.Makefile BUILD_TAG=$BUILD_TAG lint'
+                                    sh 'make -f docker.Makefile BUILD_TAG=$BUILD_TAG cross e2e-cross tars'
+                                    dir('bin') {
+                                        stash name: 'binaries'
+                                    }
+                                    dir('e2e') {
+                                        stash name: 'e2e'
+                                    }
+                                    dir('examples') {
+                                        stash name: 'examples'
+                                    }
+                                    if(!(env.BRANCH_NAME ==~ "PR-\\d+")) {
+                                        stash name: 'artifacts', includes: 'bin/*.tar.gz', excludes: 'bin/*-e2e-*'
+                                    }
+                                    archiveArtifacts 'bin/*.tar.gz'
+                                } finally {
+                                    def clean_images = /docker image ls --format "{{.ID}}\t{{.Tag}}" | grep $BUILD_TAG | awk '{print $1}' | xargs docker image rm -f/
+                                    sh clean_images
+                                }
                             }
-                        } finally {
-                            def clean_images = /docker image ls --format "{{.ID}}\t{{.Tag}}" | grep $(git describe --always --dirty) | awk '{print $1}' | xargs docker image rm -f/
-                            sh clean_images
                         }
                     }
-                }
-            }
-            post {
-                always {
-                    deleteDir()
-                }
-            }
-        }
-        stage('Test') {
-            parallel {
-                stage("Coverage") {
```

Comment:
This dissapears here. Was it deliberate? (I think so, I think it's, and was already, covered by the other Jenkins, but want to double check)

Reply:
Fixed!